CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ Changes since v2.13
 - Experimental permissions API that produces GA4GH Visas is now documented in [docs/ga4gh-visas.md](docs/ga4gh-visas.md)
 - OIDC scopes are configurable via `:oidc-scopes`. See [docs/configuration.md](docs/configuration.md).
 - REMS now reads GA4GH Passports on login and stores the ResearcherStatus of the user. See [docs/ga4gh-visas.md](docs/ga4gh-visas.md). (#2124)
+- Automated accessibility test report using [əxe](https://www.deque.com/axe/) (#2263)
```

Comment:
do they really style it as əxe? I only see axe on the site.

Reply:
Fixed